include/sys/dmu_send.h

```diff
@@ -76,4 +76,70 @@ int dmu_recv_stream(dmu_recv_cookie_t *drc, struct vnode *vp, offset_t *voffp,
 int dmu_recv_end(dmu_recv_cookie_t *drc, void *owner);
 boolean_t dmu_objset_is_receiving(objset_t *os);
 
+#define	dprintf_drro(drro, err) do { \
```

Comment:
Done. Made one dprintf wrapper function that handles the various drr record types.